Cargo.toml

```diff
@@ -83,7 +83,8 @@ members = [
     "utils/call-gen",
     "utils/utils",
     "utils/bags-thresholds",
-    "utils/integral-tests",
+    # TODO: Remove gov deps
```

Comment:
Isn't solved yet?

Reply:
onto it shortly

Reply:
Part of this tests has been moved to staking-rewards pallet. The rest should be rewritten by @ekovalev later.